src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -480,7 +480,7 @@ public void InvokeCatchingRelevantIOExceptions(Action action, Action<Exception>
             }
         }
 
-        private HashSet<Skimmer<TContext>> CreateSkimmers(TContext context)
+        protected virtual HashSet<Skimmer<TContext>> CreateSkimmers(TContext context)
```

Comment:
>protected virtual [](start = 8, length = 17)

I originally thought I would need to override this method in order to control skimmer construction from tests. In the event, did not need this. This method should already have been overridable though.